source/recovery.rst

```diff
@@ -19,35 +19,30 @@ continuously restarting, and generates the following errors in the log::
 The Percona Operator for Percona XtraDB Cluster provides two ways of recovery
 after a full cluster crash.
 
-* The automated :ref:`recovery-bootstrap` is the simplest one, but it
-  may cause loss of several recent transactions.
-* The manual :ref:`recovery-object-surgery` includes a lot of operations, but
-  it allows to restore all the data.
+The Operator is able to provide automatic crash recovery starting from version
+``1.7``. For previous Operator versions, crash recovery can be done
+:ref:`manually<recovery-object-surgery>`.
 
-.. _recovery-bootstrap:
+.. _recovery-auto:
 
-Bootstrap Crash Recovery method
--------------------------------
+Automatic Crash Recovery
+------------------------
 
-In this case recovery is done automatically. The recovery is triggered by the
-``pxc.forceUnsafeBootstrap`` option set to ``true`` in the ``deploy/cr.yaml``
-file::
+Recovery is done automatically if the ``spec.autoPXCRecovery`` option set to
```

Comment:
It is by default 'true', even if this option does not exist or is commented in CR.

Reply:
And yes, not it is `spec.pxc.autoRecovery`

Reply:
Ok, so pxc.autoRecovery is true by default. 
Currently, I see ``#    autoRecovery: false`` in deploy/cr.yaml. Should it be fixed there?